cmd/konvoy-image/cmd/artifacts.go

```diff
@@ -0,0 +1,33 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/mesosphere/konvoy-image-builder/pkg/app"
+)
+
+var artifactsFlags app.ArtifactsCmdFlags
+
+var artifactsCmd = &cobra.Command{
+	Use:   "artifacts",
+	Short: "upload artifacts to hosts defined in inventory-file",
+	Args:  cobra.NoArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		err := app.UploadArtifacts(artifactsFlags)
+		if err != nil {
+			return fmt.Errorf("failed to upload artifacts %w", err)
+		}
+		return nil
+	},
+}
+
+func init() {
+	fs := artifactsCmd.Flags()
+	fs.StringVar(&artifactsFlags.Inventory, "inventory-file", "inventory.yaml", "an ansible inventory defining your infrastructure")
```

Comment:
we should add shorthands notations to all commands. wdyt?

Reply:
These should probably be addressed in a refactor for all of the subcommands.

Reply:
sounds good.

Reply:
if we are following kubectl conventions, it would be good to follow those when we refactoring flags. 
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-cli/kubectl-conventions.md#flag-conventions